### PR TITLE
Use RPA_SELENIUM_BROWSER_ORDER env variable to determine default browsers

### PR DESCRIPTION
**Pull request by avaissi**
* ~~ROBOCORP_BROWSERS~~ RPA_SELENIUM_BROWSER_ORDER env variable, if available, is used to determine default browsers
* ~~ROBOCORP_BROWSERS~~ RPA_SELENIUM_BROWSER_ORDER env is re-checked each time Open Available Browser is called

### CODE REVIEW

**Comment by avaissi** [comment]
The reasoning behind the PR is give control for user to determine the order default browsers are tried

**Review by cmin764 — COMMENTED** [review]
This looks good in general! An additional improvement would be to move the browser preference out of the _core_ package entirely.

Is this old `DRIVER_PREFERENCE` used in any other place apart from `RPA.Browser.Selenium`? If not, then maybe it would be best to move this logic near the library itself in _main_. What do you think?

**Comment by avaissi** [comment]
> This looks good in general! An additional improvement would be to move the browser preference function out of the _core_ package entirely, into _main_.
> 
> Is this old `DRIVER_PREFERENCE` constant used in any other place apart from `RPA.Browser.Selenium`? If not, then maybe it would be best to move this logic near the library itself in _main_ (so we don't depend on _core_ updates for such behaviour). What do you think?
> 
> Newly created [Issue](https://github.com/robocorp/rpaframework/issues/781) from [epic](https://github.com/robocorp/rpaframework/issues/720)

This sounds good to me. Less dependencies the better 👍 I'll update the PR along with possible variable name change

**Review by cmin764 — CHANGES_REQUESTED** [review]
This looks better now!

Further required changes (which I guess they are already WIP since this PR is still marked as draft):
1. Bump the major.minor.**hotfix** version of _core_, even if it looks like a breaking change, since only _main_ depends on this change and is already made independent of it in this PR.
2. Bump the hotfix version as well of _main_.
3. Add release note into for main into _docs/source/releasenotes.rst_.
4. You can merge.

### Post-merge
1. Release _core_ from master.
2. Poetry update _main_ (to use latest just released _core_) and release _main_.
3. Release published with releasenotes.io and Slack announcement channels. (with release notes taken from rpaframework.org)

File: docs/source/releasenotes.rst [hunk 1]
```diff
@@ -9,6 +9,12 @@ Release notes
 `Released <https://pypi.org/project/rpaframework/#history>`_
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+20.1.1 - 25 Jan 2023
+--------------------
+
+- Library **RPA.Browser.Selenium** (:issue:`781`): Ability to determine the
+  default search order of browsers, when using the ``Open Available Browser``-keyword
```
Comment (cmin764) [inline docs/source/releasenotes.rst:16]:
```suggestion
  default search order of browsers, when using the ``Open Available Browser`` keyword.
```

maybe?

File: packages/main/src/RPA/Browser/Selenium.py [hunk 2]
```diff
@@ -64,6 +64,28 @@ def ensure_scheme(url: str, default: Optional[str]) -> str:
...
+def browser_preferences() -> Dict[str, List[str]]:
+    """Get lists of browser preferences for OS.
+    Prefers environment variable RPA_SELENIUM_BROWSER_ORDER, if defined.
+    """
+    browsers = os.getenv("RPA_SELENIUM_BROWSER_ORDER", "")
+    if browsers:
+        preferences = {
+            "default": [
+                browser.strip()
+                for browser in os.getenv("RPA_SELENIUM_BROWSER_ORDER", "").split(sep=",")
```
Comment (cmin764) [inline packages/main/src/RPA/Browser/Selenium.py:76]:
```suggestion
                for browser in browsers.split(sep=",")
```
DRY

**Review by cmin764 — APPROVED** [review]
LGTM! (left some nits)

**Pull request merged**